### DISCUSSION TITLE
918+

### DISCUSSION THREAD

**Discussion by dainosan** [007revad/Synology_HDD_db#365] [General]
at first I thought it was not possible so thumb UP however after doing this in RAID0 on a 918, 2 M2 of 256 GB I noticed how it is simply not as fast as I thought it would be 

dd if=/dev/zero of=/volume3/tempfile bs=5M count=64 conv=fdatasync
64+0 records in
64+0 records out
335544320 bytes (336 MB, 320 MiB) copied, 1.61327 s, 208 MB/s

anything I need to do ?

**Comment by 007revad** [comment]
I believe the DS918+ M.2 slots are PCIe 2.0 x1. The main benefit is I/O is faster than HDDs.

What do the following commands return?
```
sudo hdparm -tT --direct /dev/nvme0n1
sudo hdparm -tT --direct /dev/nvme1n1
```

The M.2 slots on my DS720+ are also PCIe 2.0 x1 and I can only get 390 MB/s. 
See https://github.com/007revad/Synology_Information_Wiki/blob/main/pages/NVMe-speed.md

**Comment by dainosan** [comment]
thank you for your prompt response. i have to say that magically about 2/3 hours that dd test it started to pick up, this morning seems fast. i wonder if some process was running in the background

sudo dd if=/dev/zero of=/volume3/tempfile bs=5M count=64 
64+0 records in
64+0 records out
335544320 bytes (336 MB, 320 MiB) copied, 0.373552 s, 898 MB/s

sudo dd if=/dev/zero of=/volume3/tempfile bs=5M count=64
64+0 records in
64+0 records out
335544320 bytes (336 MB, 320 MiB) copied, 0.356143 s, 942 MB/s
sudo dd if=/dev/zero of=/volume3/tempfile bs=5M count=64
64+0 records in
64+0 records out
335544320 bytes (336 MB, 320 MiB) copied, 0.353994 s, 948 MB/s

sudo hdparm -tT --direct /dev/nvme0n1

/dev/nvme0n1:
 Timing O_DIRECT cached reads:   800 MB in  2.00 seconds = 399.68 MB/sec
 Timing O_DIRECT disk reads: 1218 MB in  3.00 seconds = 405.91 MB/sec

sudo hdparm -tT --direct /dev/nvme1n1

/dev/nvme1n1:
 Timing O_DIRECT cached reads:   790 MB in  2.00 seconds = 394.87 MB/sec
 Timing O_DIRECT disk reads: 1202 MB in  3.00 seconds = 400.65 MB/sec
